vllm/engine/async_llm_engine.py

```diff
@@ -348,6 +349,12 @@ def from_engine_args(
                 "Ray is not supported with the CPU backend.")
             from vllm.executor.cpu_executor import CPUExecutorAsync
             executor_class = CPUExecutorAsync
+        elif engine_config.device_config.device_type == "xpu" and is_xpu():
+            if (engine_config.parallel_config.worker_use_ray):
+                logger.warning("not support ray yet")
```

Comment:
Shouldn't we do `assert not engine_config.parallel_config.worker_use_ray` instead?